sound/soc/sof/topology.c

```diff
@@ -2188,6 +2188,11 @@ static int sof_link_dmic_load(struct snd_soc_component *scomp, int index,
 			ipc_config->dmic.pdm[j].skew);
 	}
 
+	/* TODO: This is for backward compatibility and can be removed
+	 * later when ABI major version is incremented.
+	 */
+	ipc_config->dmic.fifo_bits_b = 16;
+
```

Comment:
It's just simpler to do a test with 
````c
if (ABI version < 3.0.1) {
    /* this takes care of backwards compatible handling of fifo_bits_b */
    ipc_config->dmic.reserved_2 = 16;   
}
````

and mark reserved2 in the header as being used for < 3.0.1

Reply:
Thanks, I'll change it.

Reply:
@plbossart I just pushed an updated PR with this fix.